libretro/Makefile.common

```diff
@@ -699,11 +703,7 @@ ifneq (,$(findstring $(GIT_VERSION),$(shell grep -s char $(GIT_VERSION_SRC))))
 endif
 
 ifneq ($(GIT_VERSION_NO_UPDATE),1)
-   $(shell echo '// This is a generated file.' > $(GIT_VERSION_SRC))
-   $(shell echo >> $(GIT_VERSION_SRC))
    $(shell echo 'const char *PPSSPP_GIT_VERSION = "${GIT_VERSION}";' >> $(GIT_VERSION_SRC))
-   $(shell echo >> $(GIT_VERSION_SRC))
-   $(shell echo "// If you don't want this file to update/recompile, change to 1." >> $(GIT_VERSION_SRC))
```

Comment:
What was the reason to remove this?

-[Unknown]

Reply:
Huh yeah good question. I just went ahead merged because it's in a retro specific file but I don't see a reason to remove this either.